### ISSUE TITLE
Improve reco-sim vertex association for high PU events with timing

### ISSUE THREAD

**Issue by perrotta** [cms-sw/cmssw#21167]
Following the discussion in #20709, it turned out that reco-sim 4D vertex association must be improved to cope with the high PU.

A dedicated PR is expected, which will resolve this issue

@lgray @makortel

**Comment by perrotta** [comment]
assign dqm

**Comment by perrotta** [comment]
assign upgrade

**Comment by dmitrijus** [comment]
+1

Why was DQM assigned? I see no reason....

**Comment by kpedro88** [comment]
@dmitrijus this issue relates to tracking validation. Please read the discussion in #20709 before blindly signing off on an incomplete issue.

**Comment by dmitrijus** [comment]
-1

**Comment by slava77** [comment]
@dmitrijus @kpedro88 
was there some progress following this issue?

@dmitrijus  perhaps to keep this on dqm list (rather than "dqm-rejected") you could edit your "-1" and drop it so that it appears on your active list.

**Comment by kpedro88** [comment]
I haven't seen any progress on this.

**Comment by dmitrijus** [comment]
Sorry, I don't know anything about tracking validation nor it is anything we (DQM) can do ourselves.
That's why I signed (and later rejected) it so people no longer are waiting on us.

The best action to do is to either contact validation or tracking experts directly. And I see it was already done.
> @lgray @makortel

That said, I will tell @jfernan2 (DQM convener) to follow up on this issue.
